### ISSUE TITLE
Back Button Doesn't Work

### ISSUE THREAD

**Issue by BahadirKaya90** [AppGround-io/bluetooth-keyboard-and-mouse-support#78]
1. The Back Button Doesn't Work in each mouse/keyboard and Multimedia sections. The screenshots are attached. I use Xiaomi Mi 8 Android 10 and the recepient device is Android 8 Philips 49PUS7502. The Back button was working on 2.5.0 version. I just updated to the last version and it doesn't work. 

2. Other issue is the that It was perfectly connecting on new devices when the application has "Compatibility Mode" Now it never connects to a new device!!! Why did you remove compatibility mode! I had to find 2.5.0 version from Google and with that I paired it to my TV and the updated to new version later. 

![Screenshot_2020-09-20-14-50-45-199_io appground blek](https://user-images.githubusercontent.com/52353519/93710821-064fbb00-fb52-11ea-9b54-701b25fabd16.jpg)

![Screenshot_2020-09-20-14-51-42-565_io appground blek](https://user-images.githubusercontent.com/52353519/93710813-fd5ee980-fb51-11ea-908f-7b49f0a1efd1.jpg)

**Comment by AppGround-io** [comment]
The back button in the current version does not work with the connection from an old version. You have to create the connection from the new version for the back button to work. 

The compatibility mode is automatically enabled when the device does not support the standard connection mode. Your device supports the standard connection mode. If it does not work, remove the Bluetooth connection from both device and pair theme again when the app is open.

**Comment by BahadirKaya90** [comment]
I removed the app totally and then installed the latest version again. It is paired and connected without problem but the back button is still not working.

**Comment by AppGround-io** [comment]
Did you remove the Bluetooth connection on the TV as well?

**Comment by ShiverZm** [comment]
my handsome bro, would you mind telling me what is the usage code of the back/home/menu in hid,i find for a long time,don care ,i just develop hid device.

**Comment by BahadirKaya90** [comment]
Sorry for very late answer. I wasn't using the app since the back button doesn't work. I wanted to give it a try again but still the back button doesn't work.

I uninstalled the previous version apk from mobile and removed bluetooth connection from Android tv and tryed to repair the devices but my android tv doesn't see the new version. And there is no compatibility mode on the new versions like previous ones.

On the previous ones I was switching to compatibility mode and my TV was finding in seconds and connecting without any problem. But on new versions my TV doesn't see the device since there is no connectivity mode. 

I had to remove the app again.
